docs/DeveloperGuide.adoc

```diff
@@ -723,31 +723,73 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person |key data into my app
+
+|`* * *` |user |edit an existing person |change details that are no longer pertinent
+
+|`* * *` |user |have a list of intelligent autocomplete options displayed when I make a command|type commands quicker and easier
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |secure user |secure my app with a password when not in use |prevent unauthorized access into my app
 
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |user |delete a specific tag |remove tags that I no longer need
+
+|`* *`|user |store notes about a contact|remember things pertaining to the contact
+
+|`* *` |user |use abbreviation when typing commands |type commands faster
+
+|`* *` |user |be prompted for confirmation when clearing the address book |avoid accidentally clearing the address book
+
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |user with many persons in the address book |delete a person using their name |remove a contact without having to find their index
+
+|`*` |advanced user |amend the storage file manually |import and export data easily
+
 |=======================================================================
 
-{More to be added}
+[discrete]
+=== Completed user stories
```

Comment:
we will have a different mechanism for tracking completed user stories. no need to record it here like you have done now.
no harm though!

Reply:
O.o How?

Reply:
issue tracker that was announced this week!

Reply:
Migrated